### ISSUE TITLE
[feature] : Add shortcut to open devtools (CMD/Ctrl + Shift + I/J)

### ISSUE THREAD

**Issue by bpoulaindev** [usebruno/bruno#1360]
Add a global keyboard shortcut to open Devtools.
Also useful for the contributor experience when needing to log something while having an error screen.

**Comment by daltonrowe** [comment]
`Command + Option + I` works open Chrome Dev Tools on Mac (same as on Chrome).
@bpoulaindev Is this what you're looking for or referring to something else?

**Comment by bpoulaindev** [comment]
@daltonrowe Yeah my bad i was pressing the wrong key lol, i'm closing the issue
